integration-tests/gradle/src/test/resources/basic-java-platform-module/build.gradle

```diff
@@ -18,13 +18,13 @@ allprojects {
 
 subprojects{
      repositories {
+          mavenCentral()
```

Comment:
Ah this reminds me of something I wanted to ask you anyway: In CI we use `.github/mvn-settings.xml` for Maven, which uses the Google EU central mirror, but I have seen Gradle tests downloading stuff from central instead.
So I guess something is missing to pass it to Gradle in CI?

Reply:
Actually, I don't know, if this is a central mirror, I think this could be ok. I will add it too, in case of `mavenCentral()` failure, it will be used.